examples/twittermap/web/conf/application.conf

```diff
@@ -115,6 +115,12 @@ us.city.path = "/public/data/city.sample.json"
 
 tweet.url = "http://twitterMapIndex-search-proxy.herokuapp.com/search/tweets?q=%s"
 
+# Heatmap parameters
+# How many recent days of the time range predicate of each heatmap query.
+# heatmap.samplingDayRange = 30
```

Comment:
Let's also mention that 30 is the default value.

Reply:
Done